plugins/woocommerce/client/legacy/js/frontend/checkout.js

```diff
@@ -561,9 +561,7 @@ jQuery( function( $ ) {
 						wc_checkout_form.detachUnloadEventsOnSubmit();
 
 						wc_checkout_form.submit_error(
-							'<div class="woocommerce-error">' +
-							( errorThrown || wc_checkout_params.i18n_checkout_error ) +
-							'</div>'
+							'<div class="woocommerce-error">' + wc_checkout_params.i18n_checkout_error + '</div>'
```

Comment:
Wondering if we should remove the unused parameters from `error:	function( jqXHR, textStatus, errorThrown ) {` ?

Also, is it worth double checking that `i18n_checkout_error` exists on `wc_checkout_params` before trying to access it?

Reply:
If it doesn't exist (it should though), do you think we should we still output the cryptic `errorThrown`? Got have something. If no regressions are introduced, `i18n_checkout_error` should always be available, turning the chance of a technical error outputting zero

Reply:
Yeah, I think so. In case (for whatever reason) the `i18n_checkout_error ` is not set, `errorThrown` is an OK fallback.

Reply:
> Also, is it worth double checking that i18n_checkout_error exists on wc_checkout_params before trying to access it?

I see other places where we're accessing it directly, but I added the safeguard nevertheless!
Gonna try and make testing instructions simpler before requesting another review 👍🏼